setup.cfg

```diff
@@ -59,7 +59,14 @@ exclude=.md,
     *_pb2.py,
     autonomy/__init__.py,
     custom_types.py,
-    *_pb2_grpc.py
+    *_pb2_grpc.py,
+    packages/valory/connections/http_client,
+    packages/valory/connections/ledger,
+    packages/valory/connections/p2p_libp2p_client,
+    packages/valory/protocols/acn,
+    packages/valory/protocols/contract_api,
+    packages/valory/protocols/http,
+    packages/valory/protocols/ledger_api
```

Comment:
open-aea has different versions of isort and black so we skip these packages from checks